src/components/Footer/Footer.jsx

```diff
@@ -1,5 +1,6 @@
 import OptimizedSiteLink from "./OptimizedSiteLink";
 import OPTIMIZED_SITE_LIST from "../../utils/optimizedSiteList";
+import GradientPatchCheckIcon from "../shared/GradientPatchCheckIcon";
```

Comment:
```suggestion
import OptimizedSiteLink from "./OptimizedSiteLink";
import GradientPatchCheckIcon from "../shared/GradientPatchCheckIcon";

import OPTIMIZED_SITE_LIST from "../../utils/optimizedSiteList";
```

컴포넌트와 utils를 구분하면 좋을 것 같습니다.